validation/superchain-version_test.go

```diff
@@ -42,21 +42,20 @@ func TestSuperchainWideContractVersions(t *testing.T) {
 
 func TestContractVersions(t *testing.T) {
 	isExcluded := map[uint64]bool{
-		291:          true,
```

Comment:
There are more tests which can be unskipped now, for example the `validation/resource-config_test.go` test.